libs/cms/src/lib/models/genericList.model.ts

```diff
@@ -30,6 +30,9 @@ export class GenericList {
 
   @CacheField(() => GetGenericListItemsInputOrderBy, { nullable: true })
   defaultOrder?: GetGenericListItemsInputOrderBy
+
+  @CacheField({ nullable: true })
```

Comment:
1. This doesn't have to be a cache field
2. Shouldn't we add the GraphQL type to this field? () => Boolean, {nullable: true}